server.js

```diff
@@ -14,14 +14,25 @@ try {
   pg = require("pg");
 }
 
-var paypal_query = "SELECT SUM(settle_amount)::numeric FROM paypal WHERE timestamp > $1 AND timestamp < $2 " +
-                   "AND ((type IN ('Donation', 'Payment') AND status = 'Completed') " +
-                   "OR type = 'Temporary Hold');";
-var stripe_query = "SELECT SUM(settle_amount)::numeric FROM stripe WHERE refunded = '0.00' " +
-                   "AND status = 'succeeded' AND timestamp > $1 AND timestamp < $2;";
-var bycountry_query = "SELECT country_code, sum(amount)::numeric, count(*) FROM paypal " +
-                      "WHERE timestamp > $1 AND timestamp < $2 AND country_code IS NOT NULL " +
-                      "GROUP BY country_code;";
+var paypal_total_query = `SELECT SUM(settle_amount)::numeric FROM paypal WHERE timestamp > $1 AND timestamp < $2
+                          AND ((type IN ('Donation', 'Payment') AND status = 'Completed')
+                          OR type = 'Temporary Hold');`;
+var stripe_total_query = `SELECT SUM(settle_amount)::numeric FROM stripe WHERE refunded = '0.00'
+                          AND status = 'succeeded' AND timestamp > $1 AND timestamp < $2;`;
+
+var bycountry_query = `SELECT country_code, sum(total)::numeric AS total, sum(donors) AS donors FROM (
+                       SELECT country_code, sum(amount)::numeric AS total, count(*) AS donors FROM paypal
+                       WHERE timestamp > $1 AND timestamp < $2
+                       AND ((type IN ('Donation', 'Payment') AND status = 'Completed')
+                       OR type = 'Temporary Hold') AND country_code IS NOT NULL
+                       GROUP BY country_code
+                       UNION
```

Comment:
I think this should be `UNION ALL` on the very slight chance that the number and value of donors for a given country in Paypal and Stripe was the same.  In that scenario, using `UNION` would remove one of those as a duplicate, but with `UNION ALL` it would include both and they would then be summed up correctly.